lua/autorun/server/sv_restricted_propcore_functions.lua

```diff
@@ -0,0 +1,64 @@
+-- Restricted Propcore Functions
+---------------------------------------------------------------------------------------
+local RESTRICTED_FUNCTIONS = {
+  'propSpawn(sn)',
+  'propSpawn(en)',
+  'propSpawn(svn)',
+  'propSpawn(evn)',
+  'propSpawn(san)',
+  'propSpawn(ean)',
+  'propSpawn(svan)',
+  'propSpawn(evan)',
+  'seatSpawn(sn)',
+  'seatSpawn(svan)',
+  'setPos(e:v)',
+  'propManipulate(e:vannn)',
+  'reposition(e:v)'
+}
+---------------------------------------------------------------------------------------
+
+-- Get a copy of the provided table
+local function copyTableByValue(tab)
+    returnTable = {}
+
+    -- Copy by value
+    for key, value in pairs( tab ) do
+        returnTable[key] = value
+    end
+
+    return returnTable
+end
```

Comment:
I've confirmed that this works.
The tables, when assigned by `=`, were being assigned by reference, not value.

Not anymore!